### ISSUE TITLE
Rancher Desktop crash on openSUSE Tumbleweed

### ISSUE THREAD

**Issue by rak-phillip** [rancher-sandbox/rancher-desktop#749]
From what I can gather, this seems to be an issue with electron and `glibc`; this doesn't seem to be an issue experienced exclusively with openSUSE, but it's the environment where I uncovered it.

I'm currently running `glibc: 2.34`. It looks like updating to electron 14+ will resolve the problem.

Related issues from other projects:

https://bugs.launchpad.net/ubuntu/+source/glibc/+bug/1944468
https://github.com/microsoft/vscode/issues/133593
https://github.com/microsoft/vscode/issues/132609

**Comment by rak-phillip** [comment]
Flatpak does run without issue, thanks for the tip! 

I still think that we should keep this issue open and push to update our electron. I still see the problem when trying to develop in Linux and the behavior and log output is very similar to the issues others have been having with apps running older versions of electron like VSCode and Teams. 

```
libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null)
libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null)
libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null)
[9177:1008/084414.640986:FATAL:gpu_data_manager_impl_private.cc(415)] GPU process isn't usable. Goodbye.
```

We can try workarounds, like passing the `--no-sandbox` argument to electron, but I'd really like the development experience to be as seamless as possible 😁

**Comment by davidcassany** [comment]
Tested RD with the new Fedora release (v35) and it suffers from the same issue as in TW, complaints about basic libraries mismatches. It only launches using the  `--no-sandbox` flag, otherwise it miserably fails.

**Comment by rak-phillip** [comment]
This issue might be resolved with #1147. I'll test again to see if it can be reproduced on my end.

**Comment by evertonlperes** [comment]
> This issue might be resolved with #1147. I'll test again to see if it can be reproduced on my end.

Did a quick test and looks good :) 
The app can render and load properly, although still failing due the issue: https://github.com/rancher-sandbox/rancher-desktop/issues/1165

**Comment by rak-phillip** [comment]
Resolved in #1147
